services/cannon/src/Cannon/API.hs

```diff
@@ -39,18 +38,11 @@ import qualified Network.WebSockets          as Ws
 import qualified System.Logger.Class         as Logger
 import qualified System.IO.Strict            as Strict
 
-mkLogger :: Opts -> IO Logger
-mkLogger o = Logger.new $ Logger.defSettings
-    & Logger.setLogLevel (o^.logLevel)
-    & Logger.setOutput Logger.StdOut
-    & Logger.setFormat Nothing
-    & Logger.setNetStrings (o^.logNetStrings)
-
 run :: Opts -> IO ()
 run o = do
     ext <- loadExternal
     m <- metrics
-    g <- mkLogger o
+    g <- Logger.new $ simpleDefSettings (o^.logLevel) (o^.logNetStrings)
```

Comment:
In light of recent talks about imports; perhaps we can use `Log.simpleDefSettings` here and `Log.new` to be consistent with other files and usages in the PR?

Reply:
i made the mistake of starting with this, so now this PR will be very, very noisy...  But it's a fair point.  :)